app/decorators/hash_diff_decorator/base.rb

```diff
@@ -19,13 +19,21 @@ def to_html
     def diff_val(val)
       if val.nil?
         "[nil]"
+      elsif val.is_a?(Numeric)
+        diff_numeric(val)
       elsif val.empty?
         "[empty]"
-      elsif val.is_a?(Numeric)
-        format('%.2f', val)
       else
         val.inspect
       end
     end
+
+    def diff_numeric(val)
+      if val.is_a?(Fixnum)
+        val.to_s
+      elsif val.is_a?(Numeric)
+        format("%.2f", val)
```

Comment:
I believe `round` returns another `Numeric` (integer or float) and doesn't force 2 digits pass the decimal, which isn't what we want. We want a String always formatted like currency. 

You might expect `round(2).to_s` to work, but:
```
2.2.3 :001 > n = 123.1
 => 123.1 
2.2.3 :002 > n.round(2).to_s
 => "123.1"
```